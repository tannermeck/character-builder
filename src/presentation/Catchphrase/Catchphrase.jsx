export default function Catchphrase({ catchphraseList }){
    return (
        <>
            <h3>Catchphrases:</h3>
            {catchphraseList.map(item => {
                return <p>{item}</p>
            })}
        </>
    )
}