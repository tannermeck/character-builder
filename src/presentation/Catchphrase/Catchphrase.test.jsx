import { render } from '@testing-library/react';
import Catchphrase from './Catchphrase';

it('should render catchphrases in a snapshot', () => {
    const { container } = render(<Catchphrase catchphraseList={['woohoo', 'meow', 'bark']}/>)
    expect(container).toMatchSnapshot();
    
})