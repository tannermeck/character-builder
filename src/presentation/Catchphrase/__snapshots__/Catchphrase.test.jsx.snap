// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render catchphrases in a snapshot 1`] = `
<div>
  <h3>
    Catchphrases:
  </h3>
  <p>
    woohoo
  </p>
  <p>
    meow
  </p>
  <p>
    bark
  </p>
</div>
`;
